Makefile

```diff
@@ -0,0 +1,49 @@
+KUBECTL=$(shell which kubectl 2> /dev/null)
+CMD=kubectl
+CDI_OPERATOR_URL=$(shell curl --silent "https://api.github.com/repos/kubevirt/containerized-data-importer/releases/latest" | grep browser_download_url | grep "cdi-operator.yaml\"" | cut -d'"' -f4)
+KUBEVIRT_OPERATOR_URL=$(shell curl --silent "https://api.github.com/repos/kubevirt/kubevirt/releases/latest" | grep browser_download_url | grep "kubevirt-operator.yaml\"" | cut -d'"' -f4)
+CURRENT_CONTEXT=$(shell kubectl config current-context)
+
+ifneq (${KUBECTL},)
+	CMD=oc
+endif
+
+deploy:
+ifeq (${CMD},kubectl)
```

Comment:
I would do this in a bash script.  It's hard to do logic correctly in a Makefile.  Bash is a little better at this.  Let's try to cut these recipes down to 1 line.